packages/contentstack-bootstrap/src/bootstrap/utils.ts

```diff
@@ -123,11 +139,13 @@ const envFileHandler = async (
   let filePath;
   let fileName;
   let customHost;
+  let previewHost: string;
   const regionName = region && region.name && region.name.toLowerCase();
-  const managementAPIHost = region.cma && region.cma.substring('8');
+  previewHost = region.cda?.substring('8').replace('cdn', 'rest-preview');
   const isUSRegion = regionName === 'us' || regionName === 'na';
   if (regionName !== 'eu' && !isUSRegion) {
-    customHost = region.cma && region.cma.substring('8');
+    customHost = region.cda && region.cda.substring('8');
+    customHost = customHost.replace('cdn', 'rest-preview');
```

Comment:
Change substring input `string ('8')` to `number (8)`

Reply:
Done